vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -2214,7 +2224,8 @@ public void onClick(View v) {
 
                 if (unreadCount > 0) {
                     iconId = R.drawable.newmessages;
-                    textColor = R.color.vector_fuchsia_color;
+                    textColor = ThemeUtils.getColor(this, R.attr.vector_fuchsia_color);
```

Comment:
While I definetly agree with you that renaming these is a good idea, I would like to try and keep this PR as small as possible (to get something basic out without having tons and tons of merge conflits), so I would prefer to keep this as is for now. I would be happy to submit more follow up PR's trying to refactor the differences out of things like `vector_fuchsia_color` though. 

If you think I should go ahead anyway, let me know though! :smile: